include/Core/Users/DeleteUsersPage.php

```diff
@@ -30,12 +64,30 @@ protected function initialize() {
 		);
 
 		$this->messages = array(
-			'warning_message' => __( 'WARNING: Users deleted once cannot be retrieved back. Use with caution.', 'bulk-delete' ),
+			'warning_message' => esc_html__( 'WARNING: Users deleted once cannot be retrieved back. Use with caution.', 'bulk-delete' ),
```

Comment:
Any specific reason for changing to `esc_html__` from `__`?

Reply:
Yes, I have overridden the render_header method of BasePage in DeleteUsersPage for displaying `network level` as a link.
If you closely watch, esc_html after echo is missing in DeleteUsersPage.  That's why I have added esc_html.